### PR TITLE
chore: frontend type validation

### PR DESCRIPTION
**Pull request by ZeRego**
Closes: #4215 

### Description:

Changes:
- import blueprint types properly
- add missing properties and values
- add specific tsconfig for building + run type check on frontend build
- force `any` type for ambiguous blueprint type ( [open issue](https://github.com/palantir/blueprint/issues/5889) )

### CODE REVIEW

File: packages/frontend/src/pages/SqlRunner.styles.ts [hunk 1]
```diff
@@ -28,6 +28,7 @@ export const SqlCallout = styled(Callout)`
     margin-top: 20px;
 `;
 
+// @ts-ignore
 export const StyledTabs = styled(Tabs)`
```
Comment (IrakliJani) [inline packages/frontend/src/pages/SqlRunner.styles.ts:32]: this one says that TableProps are not exposed or something right?

File: packages/frontend/tsconfig.build.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "paths": {} // important override
```
Comment (IrakliJani) [inline packages/frontend/tsconfig.build.json:4]: ignores "common" right?

**Review by IrakliJani — APPROVED** [review]
great work! commented few questions

File: packages/frontend/src/pages/SqlRunner.styles.ts (hunk 1 above)
Comment (ZeRego) [reply packages/frontend/src/pages/SqlRunner.styles.ts:32]: yeah, blueprint types are not fully compatible with styled-components.

File: packages/frontend/tsconfig.build.json (hunk 2 above)
Comment (ZeRego) [reply packages/frontend/tsconfig.build.json:4]: Correct

**Pull request merged**